specviz/plugins/unit_change/unit_change_dialog.py

```diff
@@ -32,6 +31,12 @@ def __init__(self, *args, **kwargs):
         self.ui = loadUi(os.path.abspath(
             os.path.join(os.path.dirname(__file__),
                          ".", "unit_change_dialog.ui")), self)
+        self.spectral_axis_unit_equivalencies = []
```

Comment:
I think a future PR might want to use `self.spectral_axis_unit_equivalencies as a list of tuples (title, equivalency), rather than two lists.  It is too easy for a future programmer to add to one list and not the other and then spend lots of time looking through the code.  I don't know that this is a blocker though.

Reply:
Is that why the CI tests are failing? That is my main concern for right now.